### ISSUE TITLE
Determine pactFile URL during pactVerify

### ISSUE THREAD

**Issue by hjoliveira** [DiUS/pact-jvm#252]
Hello,

With pact-jvm-provider-gradle, is there a way to determine the URL to a pact file at runtime during pactVerify?

If not, I'm thinking a closure similar to what is available for looking up the provider host could do this:

```
pact {
    serviceProviders {
        provider1 {
            hasPactWith('consumer1') {
                pactFile = { lookupPactFile() }
            }
        }
    }
}
```

**Comment by uglyog** [comment]
Easy enough to do.

**Comment by uglyog** [comment]
This has been released with versions 3.2.7 and 2.4.9

**Comment by hjoliveira** [comment]
Amazing, thank you!
